src/hearings/models/hearings.enum.ts

```diff
@@ -206,3 +207,11 @@ export enum Mode {
   VIEW = 'view',
   VIEW_EDIT = 'view-edit'
 }
+
+export enum ControlTypeEnum {
+  TEXT_BOX,
```

Comment:
According to the [typescript guidelines](https://www.typescriptlang.org/docs/handbook/enums.html) enums should be `PascalCase` not `UPPER_CASE_SNAKE_CASE `. Minor though.

Reply:
Hi @kuda1992 and @rayliangatos  the hearings.enum has a mixture off camalCase and PascalCase could you guys decide decide which one.

Reply:
@kuda1992 where is the guideline for using PascalCase in typescript as enum? I couldn't find it on the link you provided, as it only has a single word in its example.